systemd/systemd.go

```diff
@@ -203,54 +155,51 @@ func (*systemd) LogReader(serviceNames []string, n string, follow bool) (io.Read
 
 var statusregex = regexp.MustCompile(`(?m)^(?:(.*?)=(.*))?$`)
 
-func (s *systemd) Status(serviceName string) (string, error) {
-	status, err := s.ServiceStatus(serviceName)
-	if err != nil {
-		return "", err
-	}
-
-	return status.StatusString(), nil
-}
-
-// A ServiceStatus holds structured service status information.
-type ServiceStatus struct {
-	ServiceFileName string `json:"service-file-name"`
-	LoadState       string `json:"load-state"`
-	ActiveState     string `json:"active-state"`
-	SubState        string `json:"sub-state"`
-	UnitFileState   string `json:"unit-file-state"`
-}
-
-func (status *ServiceStatus) StatusString() string {
-	return fmt.Sprintf("%s; %s; %s (%s)", status.UnitFileState, status.LoadState, status.ActiveState, status.SubState)
-}
-
-func (s *systemd) ServiceStatus(serviceName string) (*ServiceStatus, error) {
-	bs, err := SystemctlCmd("show", "--property=Id,LoadState,ActiveState,SubState,UnitFileState", serviceName)
+func (s *systemd) Status(serviceNames ...string) ([]*client.ServiceInfo, error) {
```

Comment:
If client is the wrong place, sure let's put it somewhere else. But I want to move _away_ from having duplicated structs we need to remember to keep in sync.